channel.go

```diff
@@ -1,14 +1,33 @@
 package thingfulx
 
-import (
-	"time"
-)
+import "time"
 
 // Channel represents an individual data channel being published by a Thing out
 // there in the world somewhere.
 type Channel struct {
-	ID         string    `json:"id"`
-	Value      string    `json:"value"`
+	ID           string            `json:"id"`
+	Metadata     []ChannelMetadata `json:"metadata"`
+	Observations []Observation     `json:"observations"`
+	Data         Data              `json:"data"`
+}
+
+// ChannelMetadata is an object which contains any metadata required to
+// semantically descibe a measurement collected by a Thing.
+type ChannelMetadata struct {
+	Prop string `json:"property"`
+	Val  string `json:"value"`
+}
```

Comment:
I see - so looking at the first two examples in the gist

```
 {
      "property": "thingful:hasType",
      "value": "thingful:PublicTransportVehicle"
    },
    {
      "property": "thingful:hasOperator",
      "value": "http://www.mta.info/"
    },
```

How do know to expect a url for the second one but a schema based value for the next one? 
Will there be a series of lookup tables? or is it all just shipped to neo4j for it to sort out?